src/Act/SMT.hs

```diff
@@ -191,11 +206,14 @@ mkPostconditionQueriesBehv behv@(Behaviour _ _ (Interface ifaceName decls) _ pre
     mkQuery <$> postconds
   where
     -- declare vars
-    activeLocs = locsFromBehaviour behv
```

Comment:
I'm thinking that it would be very nice if we were able to treat every location (calldata and storage) uniformly. This is related with unification of `CalldataLocation` and `StorageLocation` comment. 

For example, can we have just a single `activeLocs` that includes both calldata and storage variables that need to be declared.